src/ansys/fluent/core/utils/data_transfer.py

```diff
@@ -15,6 +15,12 @@
 network_logger = logging.getLogger("pyfluent.networking")
 
 
+class MeshWriteError(RuntimeError):
+    """Raises exception if mesh write is failed."""
```

Comment:
Similar to a previous comment, this doc string is inaccurate because the exception does not raise itself. Also, the message itself is not well structured ("is failed").

Reply:
Right we will update it similar to standard Python exception description. Thanks.